### DISCUSSION TITLE
Force email validation

### DISCUSSION THREAD

**Discussion by rabol** [jacksleight/statamic-members#20] [Q&A]
Hi

Is there a way to force the member to validate their email before login is allowed ?

Kind regards
Steen

**Comment by jacksleight** [comment]
I assume you're referring to the public register form, but want the user to validate/activate their account like they do if they're added through the control panel? That is not currently possible, and would need to be implemented in Statamic rather than Memberbox I'm afraid.

I'd suggest adding that to https://github.com/statamic/ideas, as it would be a great feature to have.

I think the only way to do it currently would be by implementing your own custom `UsersController::register` method and using that to handle the form request rather than Statamic's built-in controller. In there you could use the same code as the CP uses:

https://github.com/statamic/cms/blob/3.3/src/Http/Controllers/CP/Users/UsersController.php#L157-L160
